exercises/high-scores/canonical-data.json

```diff
@@ -0,0 +1,91 @@
+{
+  "exercise": "high-scores",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Returns the highest score",
+      "property": "highest",
+      "input": {
+        "scores": [4, 10, 7]
+      },
+      "expected": 10
+    },
+    {
+      "description": "Returns the last score",
+      "property": "latest",
+      "input": {
+        "scores": [10, 0, 9, 3]
+      },
+      "expected": 3
+    },
+    {
+      "description": "Returns the top 3 scores",
+      "cases": [
+        {
+          "description": "Returns personal bests when last is less",
+          "property": "top_3",
+          "input": {
+            "scores": [5, 3, 1]
+          },
+          "expected": [5, 3, 1]
+        },
+        {
+          "description": "Returns personal bests including zero",
+          "property": "top_3",
+          "input": {
+            "scores": [2, 0, 3]
+          },
+          "expected": [3, 2, 0]
+        },
+        {
+          "description": "Returns personal bests when there are less than 3",
+          "property": "top_3",
+          "input": {
+            "scores": [3, 7]
+          },
+          "expected": [7, 3]
+        },
+        {
+          "description": "Returns personal bests when there is only one",
+          "property": "top_3",
+          "input": {
+            "scores": [4]
+          },
+          "expected": [4]
+        },
+        {
+          "description": "Returns personal bests from a long list",
+          "property": "top_3",
+          "input": {
+            "scores": [1, 3, 9, 3, 10, 2, 1, 0, 3, 4, 4, 7, 7]
+          },
+          "expected": [10, 9, 7]
+        }
+      ]
```

Comment:
It this stray bracket causing trouble?

Reply:
Turns out it's because of `"property": "top_3",` 😛. The Readme says:

> The 'property' is a string in lowerCamelCase identifying the type of test, but most of the times it is just the name of a function being tested.

So, we're going to have a bit of trouble with the `top_3` method after all. `top3` fails too. `topThree` works. What do you think? `top` only? Something else?

Reply:
`topTriad` or `Triad` :-)  

triad: "a group or set of three related people or things."

Reply:
But also, the first thing that comes up on google under Triad:

> A triad is one of many branches of Chinese transnational organized crime syndicates based in China, Hong Kong...

Other ideas:

* `topScores`
* `topTrio` (don't like this very much either - might as well just use `topThree`)
* `leadScores`
* `best`

If you don't like any of these we can go with whatever you prefer and see if someone has better ideas when we submit the PR.

Reply:
Thanks for catching that. I'm okay with `top`.